src/Compilers/CSharp/Portable/CommandLine/CSharpCompiler.cs

```diff
@@ -288,7 +310,9 @@ internal override string GetToolName()
         /// <param name="consoleOutput"></param>
         public override void PrintHelp(TextWriter consoleOutput)
         {
-            consoleOutput.WriteLine(ErrorFacts.GetMessage(MessageID.IDS_CSCHelp, Culture));
+            consoleOutput.WriteLine(ErrorFacts.GetMessage(MessageID.IDS_CSCHelp, Culture),
+                LanguageVersion.Default.MapSpecifiedToEffectiveVersion().ToDisplayString(),
+                LanguageVersion.Latest.MapSpecifiedToEffectiveVersion().ToDisplayString());
```

Comment:
Why wold it be funny? These are just examples. Why do they have to be the two latest versions? This wouldn't make any difference, IMHO. Why do we even need the examples at all. I believe this complication is absolutely unnecessary.